src/containers/pages/OrganizationViews/SingleOrganizationView/index.tsx

```diff
@@ -0,0 +1,281 @@
+  organization: null,
+  practitioners: [],
+  serviceClass: OpenSRPService,
+};
+
+/** the interface for all SingleOrganizationView props  */
+type SingleOrgViewPropsType = SingleOrganizationViewProps & RouteComponentProps<RouteParams>;
+
+const SingleOrganizationView = (props: SingleOrgViewPropsType) => {
+  const {
+    organization,
+    practitioners,
+    serviceClass,
+    fetchOrganizationsAction,
+    fetchPractitionerRolesAction,
+  } = props;
+
+  const orgId = props.match.params.id ? props.match.params.id : '';
+
+  // functions / methods //
+
+  /** loads the organization data
+   * @param {typeof OpenSRPService} service - the opensrp service
+   * @Param {string} organizationId - the organization id
+   */
+  const loadOrganization = async (service: typeof serviceClass, organizationId: string) => {
+    const serve = new service(OPENSRP_ORGANIZATION_ENDPOINT);
+
+    serve
+      .read(organizationId)
+      .then((response: Organization) => {
+        const action = fetchOrganizationsAction([response]);
+        store.dispatch(action);
+      })
+      .catch((err: Error) => {
+        /** still don't know what we should do with errors */
+      });
+  };
+
+  /** loads the practitioners that belong to this organization
+   * @param {typeof OpenSRPService} service - the opensrp service
+   * @Param {string} organizationId - the organization id
+   */
+  const loadOrgPractitioners = async (service: typeof serviceClass, organizationId: string) => {
+    const serve = new service(OPENSRP_ORG_PRACTITIONERS_ENDPOINT);
+
+    serve
+      .read(organizationId)
+      .then((response: Practitioner[]) =>
+        store.dispatch(fetchPractitionerRolesAction(response, organizationId))
+      )
+      .catch((err: Error) => {
+        /** still don't know what we should do with errors */
+      });
+  };
+
+  /** Unassign Practitioner form organization
+   * @param {serviceClass} service - the openSRP service
+   * @param {practitionerId} practitionerId - id of practitioner
+   * @param {organizationId} organizationId - id of organization
+   */
+  const unassignPractitioner = async (
+    service: typeof serviceClass,
+    practitionerId: string,
+    organizationId: string
+  ) => {
+    const serve = new service(OPENSRP_PRACTITIONER_ROLE_ENDPOINT);
+    const payload = { organization: organizationId, practitioner: practitionerId };
+    serve.delete(payload).then(() => {
+      // remove the practitioner Role
+      loadOrgPractitioners(serviceClass, organizationId);
+    });
+  };
+
+  useEffect(() => {
+    loadOrganization(serviceClass, orgId);
+    loadOrgPractitioners(serviceClass, orgId);
+  }, []);
+
+  if (!organization) {
+    return <Loading />;
+  }
+  // props //
+
+  // props for the header breadcrumb
+  const basePage = {
+    label: ORGANIZATIONS_LABEL,
+    url: ORGANIZATIONS_LIST_URL,
+  };
+  const breadcrumbProps: BreadCrumbProps = {
+    currentPage: {
+      label: organization!.identifier,
```

Comment:
@p-netm Let's use name here instead of identifier

<img width="1099" alt="Screen Shot 2019-10-08 at 8 38 55 AM" src="https://user-images.githubusercontent.com/4141527/66410650-523c3b00-e9a7-11e9-9f16-53b3b7b721a6.png">